yktrace/src/tir.rs

```diff
@@ -126,6 +127,10 @@ impl TirTrace {
         Ok(Self { ops })
     }
 
+    pub fn op(&self, idx: usize) -> &TirOp {
+        &self.ops.get(idx).expect("bogus trace index")
```

Comment:
I think we should make this `get_unchecked` (and document that) on the basis that you should never pass an index in that you haven't checked elsewhere is safe. [We can add a `debug_assert` that checks the index is OK, but then we only pay the cost in debug mode.] Thoughts?

Reply:
Yeah. Works for me.

As long as it crashes in debug mode with some clue of where to look, I'm happy :)